doc/optimizers.md

```diff
@@ -1,3 +1,61 @@
+## AdaBelief
```

Comment:
Can you also add `AdaBelief` to the list of differentiable separable functions given in `function_types.md`?